examples/reshare/src/dkg/actor.rs

```diff
@@ -389,6 +423,20 @@ where
             }
         }
 
+        // If the initial DKG was just performed, keep running until forcible exit.
+        if self.is_dkg {
+            // Close the mailbox to prevent accepting any new messages.
+            drop(self.mailbox);
+
+            // Keep running until killed to keep the orchestrator mailbox alive, allowing
+            // peers that may have gone offline to catch up.
+            //
+            // The initial DKG process will never be exited automatically, assuming coordination
+            // between participants is manual.
+            #[allow(clippy::empty_loop)]
+            loop {}
```

Comment:
I think we can exit here and just serve blocks from marshal (and finalizations via `orchestrator`?

Maybe I'm missing something?

Reply:
Oo yeah. Thats a good suggestion, will do.